tests/integration/test_logcollector/test_macos/test_macos_format_basic.py

```diff
@@ -70,10 +67,11 @@ def test_macos_format_basic(get_configuration, configure_environment, get_connec
                                                                            macos_message['message'])
 
     elif log_command == 'os_log':
-        logcollector.generate_macos_custom_log(macos_message['type'], macos_message['subsystem'], macos_message['category'])
+        logcollector.generate_macos_custom_log(macos_message['type'], macos_message['level'],
+                                               macos_message['subsystem'], macos_message['category'])
         expected_macos_message = logcollector.format_macos_message_pattern(
                                                         'custom_log',
-                                                        logcollector.TEMPLATE_OSLOG_MESSAGE, macos_message['subsystem'],
+                                                        logcollector.TEMPLATE_OSLOG_MESSAGE, 'log', macos_message['subsystem'],
```

Comment:
Please, check PEP-8 guide style`

```
E501 line too long (127 > 120 characters)
```

Reply:
```suggestion
                                                        logcollector.TEMPLATE_OSLOG_MESSAGE, subsystem=macos_message['subsystem'],
```

Reply:
You can use the name of the argument to specify its value instead of putting the rest of the preceding arguments one by one. Python is flexible in this topic.

Also, in this case 'log' is the default value, so it is not necessary .

Reply:
https://github.com/wazuh/wazuh-qa/pull/1478/commits/e506774333b88b7ad170de4f31e650cb10f26482